### PR TITLE
Enhancement/streamline e2e test

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: run.js [hunk 1]
```diff
@@ -115,14 +115,14 @@ module.exports = function (argv, systems, cb) {
...
     };
-    var watcher = chokidar.watch('.', opts);
-
-    watcher.on('change', function (file) {
-      debug('Watcher file changed: ', file, 'restarting service:', service);
-      restartService(service, function (err) {
-        if (err) console.error('Warning: could not restart service: ' + service.name + ' - ' + err);
-      });
-    });
+    // var watcher = chokidar.watch('.', opts);
```
Comment (DanielBrierton) [inline run.js]: ........ 😄

Comment (Wardormeur) [inline run.js]: f, thought I didn't commit this one ><

**Comment by Wardormeur** [comment]
Lonely PR is SAD @DanielBrierton

**Comment by DanielBrierton** [comment]
Lonely PR can go play with its friends on `master` now 😀

**Pull request merged**